src/Microsoft.DotNet.ProjectModel/Resolution/LibraryManager.cs

```diff
@@ -170,11 +170,6 @@ public IList<DiagnosticMessage> GetAllDiagnostics()
                             continue;
                         }
 
-                        if (library.Identity.Version.IsPrerelease && !versionRange.IncludePrerelease)
```

Comment:
FYI: analogous change in NuGet here: https://github.com/NuGet/NuGet.Client/commit/93c8e25406c056aeeb80c3b8d44781d434255020